src/Components/ColaboraConNos/styles.ts

```diff
@@ -0,0 +1,66 @@
+import { StyleSheet } from "aphrodite";
+
+const styles = StyleSheet.create({
+  colaboraCard: {
+    backgroundColor: '#DDF5DB',
+    borderRadius: '10px',
+    margin: '15px',
+  },
+  colaboraContainer: {
+    backgroundColor: '#DDF5DB',
+    display: 'flex',
+    flexDirection: 'row',
+    borderRadius: '18px',
+    padding: '20px',
+    marginBottom: '0px',
+    
+  },
+  colaboraInfo: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '5px',
+  },
+  h2: {
+    color: '#000000',
```

Comment:
try to avoid using general elements css styles. It's safer to use clases.

But let's say that this works ok when you're developing the component. From that point until react makes the transpile and minification (all css.ts -> one big giant .css file) maybe this style remain like this, afecting all your h2 or p elements, leading to unwanted and propably time consuming bug

If you need you can have more than one class for an element using ``` className={css(styles.SearchButton, styles.SearchButton2, styles.SearchButton3, etc...)} ```